### DISCUSSION TITLE
When will the latest version 5.12.0 be uploaded to maven artifactory?

### DISCUSSION THREAD

**Discussion by hrach33** [swagger-api/swagger-ui#9709] [General]
When will the latest swagger-ui version (5.12.0) jar be uploaded to maven artifactory https://mvnrepository.com/artifact/org.webjars/swagger-ui?

**Comment by breun** [comment]
As fas as I know WebJars for Swagger UI are not automatically created. You can trigger the creation of a WebJar from an NPM artifact yourself on the WebJars website: https://www.webjars.org
